package.json

```diff
@@ -19,7 +19,7 @@
     "test": "node ./scripts/run-component-tests.js",
     "test-local": "yarn -s test --local",
     "test-firefox": "yarn -s test --firefox",
-    "lint": "yarn -s tslint && yarn -s stylelint && yarn -s bazel:format-lint && yarn -s ownerslint",
+    "lint": "yarn -s tslint && yarn -s stylelint && yarn -s ownerslint",
```

Comment:
We should run the ng-dev lint here to make sure that it still checks Bazel files.

Reply:
I looked at the `ng-dev` commands, but I couldn't find an equivalent.

Reply:
Hmm, is it supposed to be `ng-dev format all --check`?

Reply:
We should use `yarn -s ng-dev format changed --check`. That matches with framework

Reply:
Done.

Reply:
Just for informations sake, `ng-dev format changed` is preferred as it will only checked the changed files instead of unnecessarily checking all of the files which have gone unchanged.